t/t-migrate-export.sh

```diff
@@ -52,8 +52,9 @@ begin_test "migrate export (default branch)"
   echo "$main_attrs" | grep -q "*.md !text !filter !merge !diff"
   echo "$main_attrs" | grep -q "*.txt !text !filter !merge !diff"
 
-  [ ! $(echo "$feature_attrs" | grep -q "*.md !text !filter !merge !diff") ]
-  [ ! $(echo "$feature_attrs" | grep -q "*.txt !text !filter !merge !diff") ]
+  echo "$feature_attrs" | grep -q "*.md !text !filter !merge !diff" && exit 1
```

Comment:
Should we single quote the grep argument to avoid possible wildcard expansion (`*`) or history expansion (`!`)?

Reply:
I've opened #5915 to track the problems in our tests' use of regular expression meta-characters.